docs/kubernetes/config_examples/f5-k8s_multi-tenant-2a.yaml

```diff
@@ -0,0 +1,129 @@
+# Deploy a BIG-IP Controller instance in the k8s_test namespace
+# Manages the test partition on the BIG-IP system
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: test_k8s-bigip-ctlr
+  namespace: k8s_test
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: test_k8s-bigip-ctlr
+      labels:
+        app: k8s-bigip-ctlr
+    spec:
+      serviceAccountName: bigip-ctlr-serviceaccount
+      containers:
+        - name: test_k8s-bigip-ctlr
+          image: "f5networks/k8s-bigip-ctlr:1.3.0"
+          env:
+            - name: BIGIP_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: bigip-login
+                  key: username
+            - name: BIGIP_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: bigip-login
+                  key: password
+          command: ["/app/bin/k8s-bigip-ctlr"]
+          args: [
+            "--bigip-partition=test",
+            "--bigip-url=10.190.24.171",
+            "--bigip-username=$(BIGIP_USERNAME)",
+            "--bigip-password=$(BIGIP_PASSWORD)",
+            "--use-secrets=true",
+            "--pool-member-type=cluster",
+            ...
+            ]
+
+---
+# Deploy a BIG-IP Controller instance in the k8s_prod namespace
+# Manages the prod partition on the BIG-IP system
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: prod_k8s-bigip-ctlr
+  namespace: k8s_prod
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: prod_k8s-bigip-ctlr
+      labels:
+        app: k8s-bigip-ctlr
+    spec:
+      serviceAccountName: bigip-ctlr-serviceaccount
+      containers:
+        - name: prod_k8s-bigip-ctlr
+          image: "f5networks/k8s-bigip-ctlr:1.2.0"
+          env:
+            - name: BIGIP_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: bigip-login
+                  key: username
+            - name: BIGIP_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: bigip-login
+                  key: password
+          command: ["/app/bin/k8s-bigip-ctlr"]
+          args: [
+            "--bigip-partition=prod",
+            "--bigip-url=10.190.24.171",
+            "--bigip-username=$(BIGIP_USERNAME)",
+            "--bigip-password=$(BIGIP_PASSWORD)",
+            "use-secrets=true",
+            ...
```

Comment:
We should only provide example manifests that work 100% if a customer is to download them and then try to deploy it. Remove line 80 ellipsis as it's not needed.